pom.xml

```diff
@@ -277,6 +277,9 @@
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-javadoc-plugin</artifactId>
                     <version>3.6.0</version>
+                    <configuration>
+                        <breakiterator>true</breakiterator>
```

Comment:
This makes the Javadoc "first sentence is a summary" convention work in the presence of `i.e.` etc.